src/poller/poller-core/index.js

```diff
@@ -27,25 +27,39 @@ const monitoring = require('@google-cloud/monitoring');
 const {PubSub} = require('@google-cloud/pubsub');
 const {Spanner} = require('@google-cloud/spanner');
 const {logger} = require('../../autoscaler-common/logger');
+const {AutoscalerUnits} = require('../../autoscaler-common/types');
+
+/**
+ * @typedef {import('../../autoscaler-common/types').AutoscalerSpanner
+ * } AutoscalerSpanner
+ * @typedef {import('../../autoscaler-common/types').SpannerConfig
+ * } SpannerConfig
+ * @typedef {import('../../autoscaler-common/types').SpannerMetadata
+ * } SpannerMetadata
+ * @typedef {import('../../autoscaler-common/types').SpannerMetricValue
+ * } SpannerMetricValue
+ * @typedef {import('../../autoscaler-common/types').SpannerMetric
+ * } SpannerMetric
+ */
 
 // GCP service clients
 const metricsClient = new monitoring.MetricServiceClient();
 const pubSub = new PubSub();
 const baseDefaults = {
-  units: 'NODES',
+  units: AutoscalerUnits.NODES,
   scaleOutCoolingMinutes: 5,
   scaleInCoolingMinutes: 30,
   scalingMethod: 'STEPWISE',
```

Comment:
Worth having a type for `scalingMethod`?

Reply:
We have the pre-defined scaling methods, but custom ones can also be defined, and they are require()'d from the js files at runtime, which effectively performs the 'type checking'!

IMHO this makes it not worth doing here.

(by contrast if they were only predefined, I would recommend an enum)...